src/data/rat/defs.lean

```diff
@@ -490,9 +490,8 @@ instance : is_domain ℚ :=
   .. (infer_instance : no_zero_divisors ℚ) }
 
 /- Extra instances to short-circuit type class resolution -/
```

Comment:
This should include a remark about computability like the instances for `real` have.